src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,71 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final RegistrationService registrationService = new RegistrationServiceImpl();
+    private static User user1;
+    private static User user2;
```

Comment:
```suggestion
    private static User validUser;
    private static User invalidUser;
```

It's bad practice to use numbers in variable names